packages/engine/src/simulation/package/state/packages/behavior_execution/package.js

```diff
@@ -1,4 +1,8 @@
-// The `__i_behavior` column is reset to zero and `__behaviors` is written to
+// TODO: Propagate field specs to runners and use in state and context objects
+const BEHAVIOR_INDEX_FIELD_KEY = "_PRIVATE_14_behavior_index";
+const BEHAVIOR_IDS_FIELD_KEY = "_PRIVATE_14_behavior_ids";
+
+// The `BEHAVIOR_INDEX_FIELD_KEY` column is reset to zero and `__behaviors` is written to
```

Comment:
What did `__behaviors` refer to? Should we have changed the comment to say `_PRIVATE_14_behavior_ids` or something?

Reply:
Yeah, the comment should refer to private `behavior_ids`. I wouldn't include the 14, because that should become an automatically calculated implementation detail once we get to it.

Reply:
```suggestion
// The `BEHAVIOR_INDEX_FIELD_KEY` column is reset to zero and the private `behavior_ids`
// column is written to
```